src/main/java/net/pms/dlna/DLNAResource.java

```diff
@@ -772,6 +772,8 @@ public void addChild(DLNAResource child, boolean isNew, boolean isAddGlobally) {
 
 					if (resumeRes != null && resumeRes.media != null) {
 						resumeRes.media.setThumbready(false);
+						resumeRes.media.setMimeType(HTTPResource.VIDEO_TRANSCODE);
+						resumeRes.media.setThumbready(false);
```

Comment:
@SubJunk Why set "thumbready" twice?

Reply:
Good question, not sure how that happened. Will fix now